### DISCUSSION TITLE
Missing TracesInstance CRD in Agent Operator

### DISCUSSION THREAD

**Discussion by ashikhman** [grafana-cold-storage/agent#3643] [Q&A]
Am I missing something or there is no way to receive and send traces to Tempo using Agent Operator?

**Comment by samlaf** [comment]
Also wondering about this. Is the only way to get traces to switch to the agent?
Or is there a way to add the [Tempo configMap](https://grafana.com/docs/grafana-cloud/kubernetes-monitoring/other-methods/k8s-agent-traces/#configure-grafana-agent) as a [custom resource](https://grafana.com/docs/agent/latest/operator/add-custom-scrape-jobs/) or integration in the operator?

**Reply by winston0410** [reply]
Same. How can I send data to Tempo according to this doc?
https://grafana.com/docs/grafana-cloud/kubernetes-monitoring/other-methods/k8s-agent-traces/#configure-grafana-agent
